app/navigation/Navigator.scala

```diff
@@ -40,7 +40,7 @@ class Navigator @Inject() () {
     case TraderReferencePage         => _ => routes.UseTraderReferenceController.onPageLoad(NormalMode)
     case UseTraderReferencePage      => navigateFromUseTraderReference
     case GoodsDescriptionPage        => _ => routes.CountryOfOriginController.onPageLoad(NormalMode)
-    case CountryOfOriginPage         => _ => routes.CommodityCodeController.onPageLoad(NormalMode)
+    case CountryOfOriginPage         => _ => routes.CommodityCodeController.onPageLoadCreate(NormalMode)
```

Comment:
do we need to add one for onPageLoadUpdate method?

Reply:
i've left it for now, i think navigation will be implemented in another ticket?